### ISSUE TITLE
Bioconda channel ends at letter m

### ISSUE THREAD

**Issue by sunworshipper** [BioContainers/multi-package-containers#709]
I appears that it doesn't load the entire channel so if you search for something like: star in the bioconda channel it's not there.  The last packages start with m (which is not likely).

**Comment by bgruening** [comment]
@sunworshipper do you mean the web interface?

**Comment by sunworshipper** [comment]
@bgruening  yes.  I tried the web interface and it appears that it doesn't load the full channel.  Maybe this was tested when the channel was much smaller.

**Comment by bgruening** [comment]
Ok, thanks! ping @ypriverol :)

**Comment by ypriverol** [comment]
@sunworshipper @bgruening I will test the web. Can you let me know @sunworshipper which packages are you trying to create the multipackage

**Comment by sunworshipper** [comment]
I was looking for star.  Which is definitely in the bioconda channel.  When searches failed for the appropriate package, I narrowed down to only bioconda and realized that the last package (in an alphabetical sorted list) started with the letter m.
